src/shapes/Object/defaultValues.ts

```diff
@@ -31,10 +31,9 @@ export const cacheProperties = [
   'strokeMiterLimit',
   'backgroundColor',
   'clipPath',
-] as const;
+];
```

Comment:
this is creating a lot of warning in the build, i removed it for now.
This is also not a const, developers are free to push here what they see may fit.
TS seems to get angry if then you say is a simple array of strings

Reply:
I saw that too.
Another bad TS design